src/Serilog.Sinks.Raygun/Sinks/Raygun/NullException.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Diagnostics;
+
+namespace Serilog.Sinks.Raygun.Sinks.Raygun
```

Comment:
Nit - namespace is unhappy here; adding `<RootNamespace>Serilog</RootNamespace>` to the CSPROJ should prevent these popping up in future